examples/error-handling/src/lib.rs

```diff
@@ -0,0 +1,64 @@
+// Find all our documentation at https://docs.near.org
+use near_sdk::contract_error;
+use near_sdk::near;
+
+#[contract_error]
+pub enum MyErrorEnum {
+    X,
+}
+
+#[contract_error]
+pub struct MyErrorStruct {
+    x: u32,
+}
+
+#[near(contract_state)]
+#[derive(Default)]
+pub struct Contract {
+    value: u32,
+}
+
+#[near]
+impl Contract {
+    #[handle_result]
+    pub fn inc_handle_result(&mut self, is_error: bool) -> Result<u32, &'static str> {
```

Comment:
Let's provide doc strings that illustrate the potential return values from the RPC response point of view.

P.S. Apply the same reasoning below